herd/BPFArch_herd.ml

```diff
@@ -54,6 +64,8 @@ struct
     | X -> "*"
     | SC -> "SC"
     | N -> ""
+    | A -> "AC"
```

Comment:
You may have meant `AQ` here?

Reply:
Yes, sorry for typo. Fixed and pushed again